types/font-carrier/class/glyph.d.ts

```diff
@@ -39,3 +40,6 @@ export class Glyph extends Base<GlyphOptions> {
         }>,
     ): string;
 }
+
+export = Glyph;
+export as namespace Glyph;
```

Comment:
```suggestion
```
IMO this is CommonJS module, without support for global script in distributed package, like wth UMD modules (no code loaded via `script` tag)